app/javascript/controllers/evaluator_recusal_modal_controller.js

```diff
@@ -0,0 +1,62 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
```

Comment:
Ideally we could reuse the more general ModalController for modal management and add custom actions in the more specific controllers but I'm kinda torn on enforcing this since we've already merged a few custom modal controllers that have the same issue.